package.json

```diff
@@ -9,7 +9,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "commitlint": "./node_modules/.bin/commitlint -e",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "build:storybook": "build-storybook",
+    "deploy:storybook": "./node_modules/.bin/storybook-to-ghpages",
```

Comment:
I think you can leave out `./node_modules/.bin`, npm adds this automatically to your path at the beginning, so it's the same with and without it 👍

Reply:
I fixed this!